test/ov-to-qaut.test.ts

```diff
@@ -0,0 +1,70 @@
+import { expect, test } from 'vitest'
+import { Quaternion, Vector3 } from 'three'
+import { OrientationVector } from '../src/main'
+import { EPSILON } from '../src/constants'
+
+const ov = new OrientationVector()
+const expectedQuat = new Quaternion()
+const convertedQuat = new Quaternion()
+
+const areQuaternionsApproxEqual = (q1: Quaternion, q2: Quaternion) => {
```

Comment:
in rust, we changed this to computing the norm of the difference of quaternions and comparing to EPSILON. Could we do that here as well (also with the vector part of the orientation vector)?

Reply:
@gvaradarajan Yep, that makes sense to me! Can you point me to exactly where in rust-utils that is happening?